how-to/duplicate-project.rst

```diff
@@ -32,37 +32,30 @@ or in the project view:
 You need to give the duplicate a name, and decide whether to duplicate the project to the same organisation or a
 different organisation.
 
-From the *Copy type* select the appropriate duplication action: *Copy*, *Fork* or *Mirror*.
+Select the appropriate duplication action: *Fork* or *Mirror*.
 
+.. _duplication-types:
 
-Copy, Fork, Mirror
+Duplication types
 ~~~~~~~~~~~~~~~~~~~
 
-Copy
-^^^^
-
-A copy is the simplest form of duplication. The new project will be an exact but independent copy of the original, and
-will include its code, database content, media and environment variables.
-
-A copy preserves the Test server's Git branch only; all other branches are discarded.
-
 
 Fork
 ^^^^
 
-A fork is similar to a copy, with the distinction that the new project retains the original's Git history in its
-codebase.
+A fork is a form of duplication in which the new project will be an exact but independent copy of the original. A fork
+will include the original’s code, database content, media, environment variables and all branches,  and retains the Git
+history in its codebase. 
 
-A fork is therefore useful when you want a duplicate in order to undertake substantial new development, as it allows you
-to merge back changes from the duplicate into the original using Git.
+A fork is therefore a useful tool when you want to duplicate in order to undertake substantial new development, as it
```

Comment:
_A fork is therefore a useful tool_ - it's not really a tool. 

Maybe just:

```
A fork is therefore useful when you want to undertake substantial new development...
```

Reply:
Done